google-cloud-errorreporting/src/main/java/com/google/cloud/errorreporting/spi/v1beta1/ErrorStatsServiceSettings.java

```diff
@@ -319,14 +348,21 @@ private Builder(ErrorStatsServiceSettings settings) {
       listEventsSettings = settings.listEventsSettings.toBuilder();
       deleteEventsSettings = settings.deleteEventsSettings.toBuilder();
 
-      methodSettingsBuilders =
-          ImmutableList.<ApiCallSettings.Builder>of(
+      unaryMethodSettingsBuilders =
+          ImmutableList.<UnaryCallSettings.Builder>of(
               listGroupStatsSettings, listEventsSettings, deleteEventsSettings);
     }
 
+    private static ConnectionSettings.Builder s_getDefaultConnectionSettingsBuilder() {
```

Comment:
Is there a reason to favor these static methods over constants? Also, FYI, [google style guide](https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names) suggests that underscores should only be used for test cases.